src/com/jalasoft/search_files/search/factory_asset.py

```diff
@@ -0,0 +1,25 @@
+#import File
+import os
+from src.com.jalasoft.search_files.search.asset import Asset, File, Directory
+
+
+class FactoryAsset (object):
+
+    def __init__(self):
+        self._list_actual_directories_and_files = []
+
+    def create_asset(self, path):
+        result = []
+        for root, directories, files in os.walk(path):
+            for dir in directories:
+                directory = Directory(os.path.join(root, dir),dir, False)
```

Comment:
please add a space after the comma, just like below
directory = Directory(os.path.join(root, dir), dir, False)